### PR TITLE
Add TypoCI to the project

### LINKED ISSUE #2124: Spell Checker CI
Would we consider adding a `Spell Checker CI` for this repository?

I think that'd be a huge plus and would help us detect the typos faster.

If you've never heard about it, please check this link https://typoci.com/ out and let me know your thoughts.

### PR DESCRIPTION
**Pull request by doomspork**
This should finish the setup for TypoCI which closes #2124

### CODE REVIEW

**Comment by doomspork** [comment]
@vbrazo we will probably need to make continual changes to this file for exceptions and languages but this seems to be doing the trick.

**Comment by vbrazo** [comment]
@doomspork agree

File: .typo-ci.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - de
+  - en
+  - en_GB
+  - en_CA
+  - es
+  - fr
+  - it
+
+excluded_files:
+  - *.yml
```
Comment (doomspork) [inline .typo-ci.yml]: @vbrazo I _think_ this should handle `.typo-ci.yml`, right?

Comment (vbrazo) [reply .typo-ci.yml]: yes 👍

**Comment by doomspork** [comment]
Thanks for the suggestion @vbrazo! Depending on how well this works we can consider applying it to other projects.

**Comment by MikeRogers0** [comment]
I made Typo CI :D It's so awesome you're considering it.

If you have any thoughts/feedback I'm all ears me@mikerogers.io :)

**Comment by vbrazo** [comment]
Great initiative. We'll report bugs to you. Thanks for reaching out!

**Pull request merged**